Sage/carbon
# fix(checkbox): css styling fix for label width - Fe 2580 checkbox label

### Proposed behaviour
label width should be auto by default
![Screenshot 2020-04-13 at 18 46 07](https://user-images.githubusercontent.com/47245766/79144569-18fde100-7db7-11ea-8b6d-3cdc56ae93b1.png)


### Current behaviour
labels uses 100% width

![Screenshot 2020-04-13 at 18 46 21](https://user-images.githubusercontent.com/47245766/79144616-2e730b00-7db7-11ea-9339-ac38ee7129f6.png)


### Checklist


<del>- [ ] Cypress automation tests
<del>- [ ] Storybook added or updated
<del>- [ ] Theme support
<del>- [ ] Typescript `d.ts` file added or updated

### Testing instructions
run `npm start`
open localhost:9001
go to Experimental -> Checkbox -> Validation
Steps to reproduce:

click onto the white space onto the right from Example Checkbox (required)
Expected result:

The checkbox Example Checkbox (required) should be not checked
